plugins/katello/3.15/upgrade/index.md

```diff
@@ -72,3 +72,30 @@ For a rundown of what was added, please see the [release notes](/plugins/katello
 {% endif %}
 
 If the above steps failed, please review /var/log/foreman-installer/katello.log and [let us know about it](https://community.theforeman.org/c/support) if unable to resolve.
+
+
+## Pulp3 Migration ##
+
+Katello 3.15 introduces Pulp 3 alongside Pulp 2.  For this release only Docker and File repos will be migrated to Pulp 3 via the migration process.  This process will need to be complete before upgrading to 3.16, but can be done at any time.  Depending on how much File and Docker content is present, this can take a very long time.  The largest part of the process can be run without downtime.
+
+1.  Migrate the content:
+
+{% highlight bash %}
+foreman-maintain advanced procedure run content-prepare
+{% endhighlight %}
+
+Note that this is run without downtime, while content and repositories are migrated in the background.  This can be run multiple times.
```

Comment:
Should we recommend running it twice if it takes longer than than x minutes?

Reply:
it gets run again when you do the switchover, so i don't think that is necessary

Reply:
For ISO and File probably not. In the future it might be when people try to migrate huge amounts of data and the content changes a lot during the migration. :+1: for now.